service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/ShareCaseControllerTest.java

```diff
@@ -115,4 +214,31 @@ private MvcResult shareCase(final String caseId, String payload) throws Exceptio
                 .contentType(APPLICATION_JSON))
             .andReturn();
     }
+
+    private MvcResult shareAllCases(String localAuthority, List<String> userIds) throws Exception {
+        final JSONObject jsonObject = new JSONObject()
+            .put("localAuthority", localAuthority)
+            .put("usersIds", new JSONArray(userIds));
+        return shareAllCases(jsonObject.toString());
+    }
+
+    private MvcResult shareAllCases(String payload) throws Exception {
+        return mockMvc
+            .perform(post("/support/cases/share")
+                .content(payload)
+                .header("authorization", USER_AUTH_TOKEN)
+                .contentType(APPLICATION_JSON))
+            .andReturn();
+    }
+
+    private static CaseDetails caseDetails(String localAuthority) {
+        Map<String, Object> caseData = new HashMap<>();
+
+        Optional.ofNullable(localAuthority).ifPresent(la -> caseData.put("caseLocalAuthority", la));
```

Comment:
Would a simple if not be more readable here?

Reply:
yeah, done